app/services/search/base.rb

```diff
@@ -41,8 +41,21 @@ def document_count
         Search::Client.count(index: self::INDEX_ALIAS).dig("count")
       end
 
+      def search_documents(params:)
+        set_query_size(params)
+        query_hash = "Search::QueryBuilders::#{name.gsub('Search::', '')}".safe_constantize.new(params: params).as_hash
```

Comment:
I think `demodulize` might clean this up a little bit:

```ruby
query_hash = "Search::QueryBuilders::#{name.demodulize}".safe_constantize.new(params: params).as_hash
```

A little [trick](https://github.com/thepracticaldev/dev.to/pull/6801#discussion_r397065488) picked up from @rhymes via @citizen428 😄

Reply:
Oooooh, thats 🔥